### DISCUSSION TITLE
Music status

### DISCUSSION THREAD

**Discussion by Jorc0** [eritislami/evobot#533] [Q&A]
I would like to know the code i have to change to set the bot status current song i know its on
``  client.user.setActivity``
But i dont know the code
My bot is playing on a single server

**Comment by wolf-yuan-6115** [comment]
> I would like to know the code i have to change to set the bot status current song i know its on
>   client.user.setActivity
> But i dont know the code
> My bot is playing on a single server

Simply add these code to your bot

(include/play.js:80) :
```js
message.client.user.setActivity(`Playing ${queue.title}`, { type: "LISTENING" });
```

(include/play.js:61) :
```js
message.client.user.setActivity(`${message.client.prefix}help and ${message.client.prefix}play`, { type: "LISTENING" });
```
